lasy/utils/laser_utils.py

```diff
@@ -242,17 +241,12 @@ def get_frequency(
     is_envelope : bool (optional)
         Whether the field provided uses the envelope representation, as used
         internally in lasy. If False, field is assumed to represent the
-        electric field.
+        Hilbert transform of the electric field.
```

Comment:
Is there a reason for changing this behavior? I can explain the rationale for the original behavior: this function (and others in laser_utils) aims at being relatively general and easy-to-use: the only lasy-specific constraint is to create a `Grid` object. Asking the users to know about the Hilbert transform may be raising the barrier a bit high. For this reason, the function was able to take care of the Hilbert transform itself, and the option to pass the Hilbert transform was a way to avoid re-computing it multiple times.